MVVM Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -65,7 +65,7 @@ private class FeedLoaderStub: FeedLoader {
 
 private extension FeedViewController {
 	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
+		errorView?.show(message: errorMessage)
```

Comment:
For the same logic as for the MVP solution I recommend to display error message using the viewModel here:

```suggestion
		viewModel?.onErrorStateChange?(errorMessage)
```